backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Controllers/HomeController.cs

```diff
@@ -94,14 +94,40 @@ private async Task SetLanguageCookie()
             int userId = AuthenticationHelper.GetUserId(_httpContextAccessor.HttpContext);
             UserProfile user = await _profileService.GetUserProfile(userId);
             AntiforgeryTokenSet tokens = _antiforgery.GetAndStoreTokens(HttpContext);
-            string languageCode = ProfileHelper.GetStandardLanguageCodeForUser(user);
+            string languageCode = GetAltinnPersistenceCookieValue();
+            if (languageCode == string.Empty)
+            {
+                languageCode = ProfileHelper.GetStandardLanguageCodeForUser(user);
+            }
 
             HttpContext.Response.Cookies.Append("i18next", languageCode, new CookieOptions
             {
                 // Make this cookie readable by Javascript.
                 HttpOnly = false,
             });
         }
+        
+        private string GetAltinnPersistenceCookieValue()
+        {
+            var cookieValue = HttpContext.Request.Cookies["altinnPersistentContext"];
+
+            if (cookieValue.Contains("UL1033"))
```

Comment:
Also, I think it should be `UL=1033`
![image](https://github.com/Altinn/altinn-access-management-frontend/assets/22012997/babf7372-7460-4a8c-948b-6434b7076b87)